### ISSUE TITLE
Halting server on startup

### ISSUE THREAD

**Issue by canilsen** [shadowfacts/DiscordChat#15]
Have been using DiscordChat since you released it without issues. But today, after a scheduled restart, server would just stop responding after startup is complete.

Here is the last thing that shows up in fml log: http://pastebin.com/TQcvRLsc

Nothing has been changed with the server, or our discord, for a few days. I also tried it on a local server with same settings, and got the exact same issue. Halting on startup. 

Did something change with Discord API today maybe?

Edit: This is on 1.7.10, Forge 1558

**Comment by shadowfacts** [comment]
Hmm, looks like a JDA bug, I'll report it. Disable DiscordChat in the config.

**Comment by canilsen** [comment]
yeah I disabled the mod for now, since I got same issue on another server.

**Comment by canilsen** [comment]
I was able to get this working again, not really sure how. I tested it against a new Discord server, and saw that it worked. Went over the settings and changed a lot back to how a default server is setup. Then it worked again. I believe it must have something to do with voice permissions, as I had that disabled for my bot role. But I'm not sure. 

Either way, if anyone else has the same problem, try fiddling wth voice permissions.
